### DISCUSSION TITLE
Weird bug where it only works on one device...

### DISCUSSION THREAD

**Discussion by DodoSeal** [3kh0/ext-remover#239] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
So for some random reason it doesn't work on mine. I alert the ```chrome.management.setEnabled``` to see if it exists *(It Does)*. However, it works on my friends chromebook perfectly fine. Is there an explanation to this?

**Comment by ILikeBypassing** [comment]
LTBEEF has nothing to do with management and enrollment, LTBEEF is just a simple bookmarklet. I believe the problem is that the exploit got patched on your chromebook. ChromeOS 106 and above has LTBEEF patched, to check your chromebook version just go to chrome://version, if it it version 106 or above then it is most likely patched. If your version is not 106 or above then it could be that the administator has set a policy to block bookmarklets just for you.

**Reply by DodoSeal** [reply]
The thing is I wrote my own bookmarklets and they work fine. It just doesn't disable the extensions. Yes, I am under version 106.

**Reply by ILikeBypassing** [reply]
Maybe you copied the bookmarklet wrong? Try copy and pasting it again. If that doesn't work, it may be because you arent on a webstore page. In order for LTBEEF to work, you need to be on a chrome.google.com/webstore page. For example chrome.google.com/webstoreabc. If these don't fix it then I don't know whats wrong, but I do know a few other exploits. Depends on the extension you have currently.

**Reply by DodoSeal** [reply]
I have LanSchool

**Reply by ILikeBypassing** [reply]
ok since it redirects you, just look at the 2 methods i posted on how to crash extensions

**Reply by DodoSeal** [reply]
Where are these methods?

**Reply by ILikeBypassing** [reply]
essentially crashing the extension for a brief moment allowing you to visit the site page and spamming a bookmarklet to block redirecting
